README.md

```diff
@@ -1,73 +1,57 @@
 # Wiredcraft Back-end Developer Test
 
-Make sure you read the whole document carefully and follow the guidelines in it.
+This project is an assignment from Wiredcraft. 
 
-## Context
+It allows storing and accessing user and friend entries via a REST API.
 
-Build a RESTful API that can `get/create/update/delete` user data from a persistence database
+## Start of the project
 
-### User Model
+1. Start the PostgreSQL database
+   
+   `docker-compose up`
+   
+2. Install the dependencies
+   
+   `yarn install`
+   
+2. Start the application, will be available on port 3000
+   
+   `yarn start`
+   
+## API documentation
+
+The API documentation is available after starting at http://localhost:3000/api.
+The general routes available:
 
 ```
-{
-  "id": "xxx",                  // user ID 
-  "name": "test",               // user name
-  "dob": "",                    // date of birth
-  "address": "",                // user address
-  "description": "",            // user description
-  "createdAt": ""               // user created date
-}
+POST    /user
+GET     /user
+GET     /usr/{id}
+PATCH   /usr/{id}
+DELETE  /usr/{id}
+GET     /usr/{id}/friend
+GET     /usr/{id}/friend/nearby
+POST    /usr/{id}/friend/{otherUserId}
+DELETE  /usr/{id}/friend/{otherUserId}
 ```
 
-## Requirements
-
-### Functionality
-
-- The API should follow typical RESTful API design pattern.
-- The data should be saved in the DB.
-- Provide proper unit test.
-- Provide proper API document.
-
-### Tech stack
-
-- Use Node.js and any framework.
-- Use any DB. NoSQL DB is preferred.
-
-### Bonus
-
-- Write clear documentation on how it's designed and how to run the code.
-- Write good in-code comments.
-- Write good commit messages.
-- An online demo is always welcome.
-
-### Advanced requirements
-
-*These are used for some further challenges. You can safely skip them if you are not asked to do any, but feel free to try out.*
-
-- Provide a complete user auth (authentication/authorization/etc.) strategy, such as OAuth.
-- Provide a complete logging (when/how/etc.) strategy.
-- Imagine we have a new requirement right now that the user instances need to link to each other, i.e., a list of "followers/following" or "friends". Can you find out how you would design the model structure and what API you would build for querying or modifying it?
-- Related to the requirement above, suppose the address of user now includes a geographic coordinate(i.e., latitude and longitude), can you build an API that,
-  - given a user name
-  - return the nearby friends
-
+## Architecture
 
-## What We Care About
+This project uses the onion architecture which allows for maintainability.
+Please check out the `docs/architecture.md` for further informations.
```

Comment:
ah, I found it should be `documentation/architecture.md`

Reply:
Fixed the typo for the path name docs -> documentation